src/test/config.sol

```diff
@@ -82,10 +82,10 @@ contract Config {
         bool    offboarding;
     }
 
-    uint256 constant HUNDRED    = 10 ** 2;
-    uint256 constant THOUSAND   = 10 ** 3;
-    uint256 constant MILLION    = 10 ** 6;
-    uint256 constant BILLION    = 10 ** 9;
+    uint256 constant         HUNDRED  = 10 ** 2;
```

Comment:
```suggestion
    uint256 constant private HUNDRED  = 10 ** 2;
```

Or is there a reason for no vis declaration (can use public instead?)

I see [it isn't in dss-test](https://github.com/makerdao/dss-test/blob/52cdc7cca6093e59ec4319b8647c32db72a4f24e/src/DSSTest.sol#L47) so do we want to formalise HUNDRED?

Reply:
HUNDRED is used once for WBTC-C `dust` (Can just be 7_500 or 75_00 instead). IMO reliance on less constants / vars is useful?

Reply:
removed it